### ISSUE TITLE
Frontier: not gaining tokens in arena battles once you beat a level

### ISSUE THREAD

**Issue by SuplexCity25284** [PavelZinchenko/event-horizon-frontier-builds#356]
It was changed with the last update, that in Frontier, if you go back to an earlier, easier level in the Arena you no longer earn tokens?  Why?

I managed to get to Starbase battle 9, but get absolutely demolished by people who have Syberia already and such.  I'm only in level 220, I can't even dream of having Syberia yet.

I don't see why the tokens for earlier battles were taken away.  The entire Frontier game is a grind.  Grind to earn items, loot, money, stars.  Which is fine.

Why was that taken away in Arena?  It's one of the few places with the good items like satellites in the stores, but I can't grind tokens to earn them?

**Comment by xdeadstokex** [comment]
Temporary fix for ya,
fight here
![image](https://github.com/user-attachments/assets/f68a09bd-b75c-43f3-950e-9996a8e4da24)
find someone with a name with dash line (they are mostly weak), the one with full stat are strong (stupidly strong)
![image](https://github.com/user-attachments/assets/b5eb0a05-0e01-4a4f-bf16-bef118228e74)

**Comment by SuplexCity25284** [comment]
In starbase battle 9 I'm getting wiped out unmercifully, so I have to try something lol

**Comment by xdeadstokex** [comment]
I have just been beaten to a pulp by someone with white snake lol (their name dont have dash line), like their damage is massive, one shot my capital ship

**Comment by SuplexCity25284** [comment]
Yeah, I just had that happen in a duel.  First ship I faced was spamming like 4 or 5 Heavy Ion Cannon Mk2 and just smashed my ships.

**Comment by SuplexCity25284** [comment]
Thanks for the pointer though.  It got me past starbase battle 10 and into black bazaar 4 and 5.  Now if I could only get past the first guy in the challenge.  Their name?  Idiot.  

I wish I was kidding.

**Comment by SuplexCity25284** [comment]
Managed to get to both the challenge battle types only to get absolutely wiped out by opponent.  Which is fine, it happens.  But I also find that only one opponent name comes up.  No others.

**Comment by xdeadstokex** [comment]
@SuplexCity25284 The starbase battle is where you grind point, the challenge battle is where you grind for the leaderboard

**Comment by SuplexCity25284** [comment]
That's all well and good, but the part I found funny...and annoying was that I only had one option for a battle in either challenge, and the name of the base challenge?  Was Idiot.  I kid you not

**Comment by xdeadstokex** [comment]
@SuplexCity25284 Bug or sth, I encounted it sometimes

**Comment by SuplexCity25284** [comment]
Same here
